.github/workflows/flutterfmt.yml

```diff
@@ -9,16 +9,14 @@ on:
       - '.github/workflows/flutterfmt.sh'
       - '**.dart'
 jobs:
-
-  gofmt:
+  flutterfmt:
     name: Run flutter format
     runs-on: ubuntu-latest
     steps:
-
       - name: Install flutter
         uses: subosito/flutter-action@v2
         with:
-          flutter-version: '3.27.0'
+          flutter-version: '3.29.0'
```

Comment:
Same thought here. Though it seems like this was maybe just missed from the last PR?

Reply:
rebase on main fixed this diff, github is weird sometimes...